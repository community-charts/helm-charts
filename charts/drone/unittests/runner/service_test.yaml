suite: test drone kubernetes runner service
templates:
  - runner-service.yaml
tests:
  - it: should be empty if kube runner is not enabled
    set:
      kubeRunner.enabled: false
    asserts:
      - hasDocuments:
          count: 0

  - it: should match snapshot of default values
    set:
      kubeRunner.service.annotations:
        test-key: test-value
    chart:
      version: 1.0.0
      appVersion: 1.0.0
    asserts:
      - matchSnapshot: { }
