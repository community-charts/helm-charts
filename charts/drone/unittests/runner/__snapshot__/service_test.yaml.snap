should match snapshot of default values:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      annotations:
        test-key: test-value
      labels:
        app: drone-runner
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: runner-drone
        app.kubernetes.io/version: 1.0.0
        helm.sh/chart: drone-1.0.0
        version: 1.0.0
      name: runner-RELEASE-NAME-drone
      namespace: NAMESPACE
    spec:
      ports:
      - name: http
        port: 3000
        protocol: TCP
        targetPort: http
      selector:
        app: drone-runner
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/name: runner-drone
      type: ClusterIP
